src/Features/Lsif/Generator/Program.cs

```diff
@@ -81,21 +90,21 @@ private static async Task GenerateAsync(FileInfo? solution, FileInfo? project, F
                 if (solution != null)
                 {
                     await LocateAndRegisterMSBuild(logFile);
```

Comment:
❔ Missing cancellation token?

Reply:
nothing in that method needs CT as fat as i can tell :)